src/screens/block/block.vue

```diff
@@ -111,12 +118,26 @@ export default {
       } else if (blockHashRegex.test(this.blockId)) {
         this.$store.dispatch('loadBlock', { hash: this.blockId })
       }
+    },
+    takeAction (event) {
+      if (event.srcKey === 'right' && this.block.height < this.latestHeight) {
+        this.$router.push({name: 'Block', params: {blockId: this.block.height + 1}})
+      }
+      if (event.srcKey === 'left' && this.block.height > 1) {
+        this.$router.push({name: 'Block', params: {blockId: this.block.height - 1}})
+      }
+    },
+    getLatestBlockHeight () {
+      this.$store.dispatch('fetchHeight')
     }
   },
   mounted () {
     this.getBlock()
+    if (this.$store.state.height === 0) {
+      this.getLatestBlockHeight()
+    }
   }
 }
 </script>
 
-<style scoped src='./block.scss' lang='scss' />
+<style src='./block.scss' lang='scss' />
```

Comment:
Why `scoped` is removed?

Reply:
this happen after i get latest changes from master branch, but if script is scoped it happen that component doesn't understand css classes from external scss file